Initial revisions on the attacks section (#34)
xsleaks/wiki: content/docs/attacks/xs-search.md

## Extended Principle

While the original research around XS-Search focused on timing attacks, the principle of the attack extends to other XS-Leaks. Instead of relying on timing measurements, which are unreliable, attackers can use other XS-Leaks to achieve the same observation.

In a Query-Based Search System, a user submits queries and gets responses associated with those queries. From this action, there are two different outcomes:

1. The system shows results and the page will present a specific behavior (first state).
2. The system does not show results and the page will present a different behavior from step 1 (second state).

> If both behaviors, abstracted in the example above, can be distinguished using another XS-Leak technique, then an attacker can perform an XS-Search attack more reliably. For example, if the number of frames on a page varies based on search results (step 1 and 2 are distinguishable), this attack principle can be applied with a [Frame Counting]({{< ref "frame-counting.md" >}}) XS-Leak which is much more accurate then one using timing measurements.

Reviewer: I changed a little bit the sentence to clarify the technique a little bit. 

```suggestion
If both behaviors above can be distinguished by a more reliable XS-Leak than timing, then an attacker could perform a more efficient XS-Search attack. For example, if the number of frames on a page varies based on search results (step 1 and 2 are distinguishable), this attack principle can be applied with a [Frame Counting]({{< ref "frame-counting.md" >}}) XS-Leak which is much more accurate then one using timing measurements.
```
